aws-athena/ctas/location-access.sql

```diff
@@ -1,27 +1,28 @@
-CREATE TABLE IF NOT EXISTS location.access
-WITH (
-    FORMAT = 'Parquet',
-    WRITE_COMPRESSION = 'SNAPPY',
-    EXTERNAL_LOCATION = 's3://ccao-athena-ctas-us-east-1/location/access',
-    PARTITIONED_BY = ARRAY['year'],
-    BUCKETED_BY = ARRAY['pin10'],
-    BUCKET_COUNT = 1
-) AS (
+{{
+    config(
+        materialized='table',
+        partitioned_by=['year'],
+        bucketed_by=['pin10'],
+        bucket_count=1
+    )
+}}
+
+WITH access AS (
```

Comment:
Although the addition of this subquery is not strictly necessary, it makes the diff easier to interpret, since otherwise you would have to sift through lots of added indents. I think that's maybe possible on GitHub if you select the "hide whitespace" option...? But I leave it up to you, let me know what you'd prefer!

Reply:
Good call, issue open here! https://github.com/ccao-data/data-architecture/issues/120